sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -367,6 +367,8 @@ export type RequestBodyType = NodeJS.ReadableStream | (() => NodeJS.ReadableStre
 // @public
 export class RestError extends Error {
     constructor(message: string, options?: RestErrorOptions);
+    // (undocumented)
+    [x: symbol]: () => string;
```

Comment:
What is this symbol here and why is it new?

Reply:
It probably this, but not sure why api-extractor doesn't use the name and the ref doc

https://github.com/Azure/azure-sdk-for-js/blob/708295a1400cc256dab541a27890025f2e6253a7/sdk/core/ts-http-runtime/src/restError.ts#L91